frontend/javascripts/admin/account/account_auth_token_view.tsx

```diff
@@ -18,9 +18,14 @@ function AccountAuthTokenView() {
   }, []);
 
   async function fetchData(): Promise<void> {
-    const token = await getAuthToken();
-    setCurrentToken(token);
-    setIsLoading(false);
+    try {
+      const token = await getAuthToken();
+      setCurrentToken(token);
+    } catch (error) {
+      console.error("Failed to fetch auth token:", error);
```

Comment:
we should also alert the user with Toast.error so that they can initiate a page refresh and don't wonder about the silent failure imo.

Reply:
I pushed a commit for this.